modules/olm-bundle-format.adoc

```diff
@@ -40,7 +39,16 @@ deployment and RBAC model of the Operator.
 
 A bundle includes one ClusterServiceVersion (CSV) per directory
 and typically the CRDs that define the owned APIs of the CSV in its `manifest/`
-directory, though additional objects may be included:
+directory.
+
+The following objects can also be optionally included:
+
+* Secrets
+* ConfigMaps
+
+When these optional objects are included in a bundle, Operator Lifecycle Manager
+(OLM) can create them from the bundle and manage their lifecycle along with the
+CSV.
```

Comment:
Is it worth expanding a bit here? For example,

"When the CSV is deleted, the secret/configmap is deleted as well. When the CSV is upgraded the secret/configmap is either updated-in-place (if the name of the new one is the same) or deleted and recreated (if the name has changed between versions)."